### DISCUSSION TITLE
Android平台如何使用多路径？

### DISCUSSION THREAD

**Discussion by Yoo-O** [alibaba/xquic#444] [Q&A]
目前在Android平台集成了xquic，可以完成基本的数据收发，但是多路径无法生效，加了一些日志发现WIFI+4G情况下，4G只发送数据但是没有报文回复。

**Comment by Yanmei-Liu** [comment]
在Android平台上使用multipath需要类似demo_client中实现对应的多条路径收发包回调函数。
从问题描述上看，服务端是否收到4G路径下的报文并正常返回？如果是，则为端上的socket多路收包处理未完善；否则为发包实现有问题。

**Reply by cpp2go** [reply]
这问题解决了吗？

**Comment by cpp2go** [comment]
@Sy0307 android平台能同时启用 wifi 4g实现多路径吗？
